content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,20 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x, 1.1.x or 3.0.x
+- glibc 2.17 or newer
+
+#### Windows runtime dependencies
+
+On [Windows Microsoft Visual C++ Redistributable 2015](https://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe) or newer must be installed (which is by default the case on most modern installations).
 
```

Comment:
We're missing a paragraph about the minimum support version of Mac OS

Reply:
Please do add a suggestion for a headline and wording.